genai-perf/pyproject.toml

```diff
@@ -59,6 +59,7 @@ dependencies = [
   "pyyaml",
   "responses",
   "pillow",
+  "oputna==3.6.1",
```

Comment:
It looks like this version is unable to be found in the GitHub hook. The check is failing due to being unable to build and test as a result.

Reply:
Yes, I will need to figure this out before we merge to main, but for now I don't think this is a blocking issue (as we're only merging to a branch).

Reply:
Sounds good.